lib/gpde/n_arrays_io.c

```diff
@@ -363,7 +363,7 @@ N_array_3d *N_read_rast3d_to_array_3d(char *name, N_array_3d * array,
 
     /* Close files and exit */
     if (!Rast3d_close(map))
-        Rast3d_fatal_error(map, NULL, 0, _("Error closing g3d file"));
+        Rast3d_fatal_error(_("Error closing g3d file"));
```

Comment:
`name` seems to be available here (with the right content). Can you please change it as you did in #2597?

Reply:
I took the liberty to add this.